gladier_xpcs/flows/container_flow_base.py

```diff
@@ -8,7 +8,7 @@
 log = logging.getLogger(__name__)
 
 
-class XPCSBaseClient(GladierBaseClient):
+class ContainerBaseClient(GladierBaseClient):
```

Comment:
I'm not against it, but any reason for the rename? 

One important thing we _should_ do here is remove `retry_backoff` code below, that will conflict with the latest Gladier version v0.6.0

Reply:
renaming so I keep the "container_patch" with the same structure for all gladier-repos.